### ISSUE TITLE
Potential false positive in GRA-LOGTEMPLMISSVALS-001

### ISSUE THREAD

**Issue by nojaf** [G-Research/fsharp-analyzers#79]
From the trenches:

```fsharp
stepLog.LogWarning(sprintf "Foo %dm %ds {Wasted}\n{Result}" (int timespan.TotalMinutes) timespan.Seconds, timeWasteString, result)
```

This gives `Error GRA-LOGTEMPLMISSVALS-001 - The given values in your call to ILogger.LogWarning don't match the expected templated args` but probably shouldn't.
